cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuild.java

```diff
@@ -268,13 +300,20 @@ public Pod generateBuilderPod(boolean isOpenShift, ImagePullPolicy imagePullPoli
     protected List<Container> getContainers(ImagePullPolicy imagePullPolicy) {
         List<Container> containers = new ArrayList<>(1);
 
+        List<String> args = new ArrayList<>(4);
```

Comment:
should be this 4 + size of the additionalKanikoOptions (but only if additionalKanikoOptions != null)?
So kind of

```java
List<String> args = additionalKanikoOptions != null ? new ArrayList<>(4 + additionalKanikoOptions.size()) : new ArrayList<>(4)
```